src/pages/cli/graphql/data-modeling.mdx

```diff
@@ -549,7 +549,494 @@ type Address @model {
 ```
 
 ### Relational models with custom primary keys
-TODO
+You can define custom primary keys for relational models. The transformer will handle the foreign key fields needed by queries and mutations, which allows you to retrieve data across different relations. 
```

Comment:
"transformer" this word is not used previously in the documentation,  and "The transformer will handle the foreign key fields needed by queries and mutations" sounds more like an implementation detail. Should we expose it here?

Reply:
The "transformer" term is removed